### PR TITLE
run solitude as a proxy (bug 1112672)

### PR DESCRIPTION
**Pull request by andymckay**
* we only need to expose 2602, not the proxy
* move the environment variables to be per solitude instance, so inside supervisor.conf, not the dockerfile

### CODE REVIEW

File: bin/docker/supervisor.conf [hunk 1]
```diff
@@ -10,3 +10,18 @@ redirect_stderr=true
 stdout_logfile=logs/docker.log
 stdout_logfile_maxbytes=1MB
 stopsignal=KILL
+environment=SOLITUDE_ZIPPY_PROXY="http://solitude:2603/proxy/provider", SOLITUDE_DATABASE="mysql://root:@mysql:3306/solitude", MEMCACHE_URL="memcached:11211", SOLITUDE_URL="http://solitude:2602"
```
Comment (muffinresearch) [inline bin/docker/supervisor.conf]: Can this be wrapped without breaking the conf?

**Comment by muffinresearch** [comment]
Shiny lgtm, r+wc

File: bin/docker/supervisor.conf (hunk 1 above)
Comment (andymckay) [inline bin/docker/supervisor.conf]: they can indeed, testing it still works :)

**Pull request merged**